extra_foam/special_suite/facade.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, topic):
         self._cw = QWidget()
         self.setCentralWidget(self._cw)
 
+        # Center window
```

Comment:
Remove the comment. Function name is descriptive enough.

Reply:
Fixed in 0d6165db.